frontend/src/components/Explanation.js

```diff
@@ -1 +1,15 @@
-// This is where the explanation component will be created
\ No newline at end of file
+import '../styles/Explanation.css';
+import React from 'react';
+
+function Explanation()
+{
+    return(
+        <div class="jumbotron">
+            <h1 class="display-4">Welcome to PetFinder!</h1>
```

Comment:
Consider renaming the class names to be more aligned: lead, display-4, my-4. Do the numbers at the end mean something?